hbase-shaded-jackson-jaxrs-json-provider/pom.xml

```diff
@@ -0,0 +1,110 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <!--
+  /**
+   * Licensed to the Apache Software Foundation (ASF) under one
+   * or more contributor license agreements.  See the NOTICE file
+   * distributed with this work for additional information
+   * regarding copyright ownership.  The ASF licenses this file
+   * to you under the Apache License, Version 2.0 (the
+   * "License"); you may not use this file except in compliance
+   * with the License.  You may obtain a copy of the License at
+   *
+   *     http://www.apache.org/licenses/LICENSE-2.0
+   *
+   * Unless required by applicable law or agreed to in writing, software
+   * distributed under the License is distributed on an "AS IS" BASIS,
+   * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   * See the License for the specific language governing permissions and
+   * limitations under the License.
+   */
+
+
+    ON MVN COMPILE NOT WORKING
+
+    If you wondering why 'mvn compile' does not work building HBase
+    (in particular, if you are doing it for the first time), instead do
+    'mvn package'.  If you are interested in the full story, see
+    https://issues.apache.org/jira/browse/HBASE-6795.
+
+  -->
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.apache.hbase.thirdparty</groupId>
+        <artifactId>hbase-thirdparty</artifactId>
+        <version>3.5.1-SNAPSHOT</version>
```

Comment:
Had the same comment, otherwise lgtm

Reply:
I intended to handle the artifact version as part of a separate commit process. In part because I haven't done any releases out of this repository and I don't know the current practices.

Since you pointed this out specifically, we can discuss here.

Because this is adding a new interface without removing anything, my thinking is this would result in a bump of minor version number. I don't remove the existing `javax.ws.rs` as it was never provided by this jar directly.

Reply:
I think we're in agreement about the impact for this change. My question was if that's obviated by HBASE-25855 having landed already.

Reply:
Landing HBASE-25855 seems to have been a noop in that I didn't find any use of the typo-ed symbol in hbase-core. hbase-operator-tools and hbase-connectors are still on `hbase-thirdparty.verison=2.2.1` and don't make use of hbase-shaded-jersey at all.

Reply:
that is confusing. I'll take my release note concerns to that issue then. :)